Dockerfile.bundle

```diff
@@ -0,0 +1,36 @@
+FROM registry.ci.openshift.org/openshift/release:golang-1.15 as build
+
+ENV GO111MODULE=on
+ENV GOFLAGS=""
+
+RUN yum install -y python3-pip && \
+    yum clean all
+RUN curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | \
+    bash -s -- 3.8.8 && mv kustomize /usr/bin/
+RUN pip3 install waiting==1.4.1 mkdocs==1.1.2
+COPY --from=quay.io/openshift/origin-cli /usr/bin/oc /usr/bin
+RUN go get -u sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.0 
+RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(arch) ;; esac) \
+  && export OS=$(uname | awk '{print tolower($0)}') && export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/latest/download \
+  && curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH} \
+  && chmod +x operator-sdk_${OS}_${ARCH} \
+  && install operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
+
+COPY . .
+RUN export BUNDLE_OUTPUT_DIR=/bundle; make operator-bundle
+
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
```

Comment:
Maybe add a git_revision label as we do with all our other images?

Reply:
Done.